### PR TITLE
Update Italian create modal translations

### PR DESCRIPTION
**Pull request by slamservice**
Italian translation upgrade

### CODE REVIEW

File: packages/support/resources/lang/it/actions/create.php [hunk 1]
```diff
@@ -8,16 +8,16 @@
 
         'modal' => [
 
-            'heading' => 'Nuovo :label',
+            'heading' => 'Slava :label',
```
Comment (Simoneu01) [inline packages/support/resources/lang/it/actions/create.php]: Qui c'è un typo

File: packages/support/resources/lang/it/actions/create.php [hunk 2]
```diff
@@ -8,16 +8,16 @@
...
             'actions' => [
 
                 'create' => [
-                    'label' => 'Nuovo',
+                    'label' => 'Salva',
                 ],
 
                 'create_another' => [
-                    'label' => 'Nuovo & un altro nuovo',
+                    'label' => 'Salva & un altro nuovo',
```
Comment (Simoneu01) [inline packages/support/resources/lang/it/actions/create.php]: Qui si potrebbe mettere "Salva e crea uno nuovo "

**Review by Simoneu01 — CHANGES_REQUESTED** [review]

**Comment by zepfietje** [comment]
Could you implement the requested changes so this PR can be merged, @slamservice?

File: packages/support/resources/lang/it/actions/create.php [hunk 3]
```diff
@@ -8,16 +8,16 @@
...
```
Comment (zepfietje) [inline packages/support/resources/lang/it/actions/create.php]:
```suggestion
                    'label' => 'Salva & crea uno nuovo',
```

**Pull request merged**